Resources/public/js/views/tabs/ez-locationviewrelationstabview.js

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-locationviewrelationstabview', function (Y) {
+    "use strict";
+    /**
+     * Provides the Location View Relations Tab view class.
+     *
+     * @module ez-locationviewrelationstabview
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The Location View Relations tab class.
+     *
+     * @namespace eZ
+     * @class LocationViewRelationsTabView
+     * @constructor
+     * @extends eZ.LocationViewTabView
+     */
+    Y.eZ.LocationViewRelationsTabView = Y.Base.create('locationViewRelationsTabView', Y.eZ.LocationViewTabView, [Y.eZ.AsynchronousView], {
+        initializer: function () {
+            this._fireMethod = this._fireLoadObjectRelations;
+            this._watchAttribute = 'relatedContents';
+        },
+
+        render: function () {
+            var container = this.get('container');
+
+            container.setHTML(this.template({
+                "relatedContents": this._lookupRelationListItems(this.get('relatedContents')),
+                "loadingError": this.get('loadingError'),
+            }));
+
+            return this;
+        },
+
+        /**
+         * Provides the name of the relation type based on its Id
+         *
+         * @method _lookupRelationTypeName
+         * @protected
+         * @param {String} relationTypeId
+         * @return {String}
+         */
+        _lookupRelationTypeName: function(relationTypeId) {
+            if (relationTypeId === "ATTRIBUTE") {
```

Comment:
Shouldn't those to be constant variables? "ATTRIBUTE" string is also used in `_lookupRelationInfo` method.